### DISCUSSION TITLE
AIQuS Component

### DISCUSSION THREAD

**Discussion by Aleksandar-Elfak** [c0c0n3/kitt4sme.live#319] [Q&A] [answered]
Hello,

Where can I find the [AIQuS component](https://github.com/c0c0n3/kitt4sme/blob/master/arch/catalogue/aiqus.md) Docker image, if it is available? Additionally, I would appreciate any documentation related to this component.

Thank you in advance.

Best regards,
Aleksandar

**Comment by kostasgrevenitis** [comment, marked as answer]
Hello.

The AIQuS component docker image is not public. 

Please discuss your request directly with your mentor who will contact R2M that is the owner of the AIQuS component.
